### DISCUSSION TITLE
make a GUI version that uses tkinter

### DISCUSSION THREAD

**Discussion by kety-folf** [ThioJoe/YT-Spammer-Purge#175] [Ideas]
make a version that uses a GUI with tkinter so you just put the video url or channel url in the text  box and click a button and have the program do its thing

**Comment by ThioJoe** [comment]
how is that any different from pasting it into the console window and pressing enter

**Reply by kety-folf** [reply]
It will give it a nice look to it and for some people who don't know how to use a console all the options are right there. a console would still work as the back end but a GUI front end would make it look more user friendly.

**Reply by ThioJoe** [reply]
Tkinter looks like garbage tbh, straight out of 1995. And it would take 10x as much time to add features, without adding any real functionality. You'd paste the links into a box instead of the console, you'd click a button instead of pressing a number.

"Don't know how to use console" - They don't know how to type numbers and press enter? Cmon

**Reply by mark-IV-II** [reply]
Adding a GUI can help in avoiding the restart of the program after each run. I agree regarding Tkinter and console thing. By the way something like PyQT can redirect the whole console into the GUI and allows selecting the required files via OS native file dialogs

**Reply by kety-folf** [reply]
yep or simplepyGUI which acts as a wrapper for all the different GUI libs
